staging/src/k8s.io/kubectl/pkg/cmd/create/create_clusterrole.go

```diff
@@ -187,6 +190,11 @@ func (c *CreateClusterRoleOptions) RunCreateRole() error {
 	clusterRole.Name = c.Name
 
 	var err error
+	clusterRole.Labels, err = cmdutil.ParseLabels(c.Labels)
```

Comment:
As this is a validation that might happen on the client side, and with the approach to normalize exit codes, I think that all the cmdutil.ParseLabels should be moved to "Validate" instead of "Run", as this might be a syntax validation that happens always on the client-side.

We can expect some more feedback about this, your approach here IMO is good, the change that I'm proposing actually will have more lines as it would be:

* Add a new ParsedLabels map to be used later on the object creation
* add the flag
* Inside the Validate, populate the ParsedLabels with "cmdutil.ParseLabels(c.Labels)" if this does not return an error
* Later in Run, do a clusterRole.Labels = ParsedLabels

I know it sounds strange, and this is why I want some more opinions. While this is more "complex" it will help in a future to separate errors from client-side from errors of server-side.

But I'm also ok with this approach (I don't want to gate keep on this) and rethink in this approach in the future when exit codes normalization became something :)